ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c

```diff
@@ -3111,7 +3155,8 @@ process_vlv_index(backentry *ep, ImportWorkerInfo *info)
         struct vlvIndex *vlv_index = ps->vlv_index;
         Slapi_PBlock *pb = slapi_pblock_new();
         slapi_pblock_set(pb, SLAPI_BACKEND, be);
-        if (vlv_index && (ctx->indexAttrs==NULL || attr_in_list(vlv_index->vlv_name, ctx->indexAttrs))) {
+        if (vlv_index && vlv_index->vlv_attrinfo &&
+            is_reindexed_attr(vlv_index->vlv_attrinfo->ai_type , ctx, ctx->indexVlvs)) {
```

Comment:
It's not related to this change, but it could be nice to fix/clarify. In this function (just a few lines below) there's error message `slapi_log_err(SLAPI_LOG_ERR, "process_regular_index", "index_addordel_values_ext_sv failed.\n");`.

I think it could be a typo, as it has the wrong function name.

Reply:
Indeed! looks like a cut and paste issue! :wink: